tools/cjxl_ng_main.cc

```diff
@@ -235,37 +216,23 @@ DEFINE_int32(photon_noise, 0,
 DEFINE_int32(codestream_level, 5, "The codestream level. Either `5` or `10`.");
 
 DEFINE_double(
-    distance, 1.0,  // TODO(tfish): wire this up.
-    "Max. butteraugli distance, lower = higher quality. Range: 0 .. 25.\n"
+    distance, 1.0,
+    "Max. butteraugli distance, lower = higher quality.\n"
     "    0.0 = mathematically lossless. Default for already-lossy input "
     "(JPEG/GIF).\n"
```

Comment:
I think we still have to make sure that this actually happens: doing lossless when the input is JPEG or GIF. I think we currently just load the input and use d1 regardless of what the input format was.

Reply:
Done.